android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java

```diff
@@ -71,7 +72,11 @@ public static Session createSessionFromCurrentSession() {
 	}
 
 	public static boolean isLoggedIn() {
-		return currentUserSession != null;
+		if (currentUserSession == null) {
+			LiferayLogger.e("You need to be logged to view the content!");
```

Comment:
In most of the cases this method is used when the app is going to load some content or is going to move you to the next activity and it's necessary to be logged in. It's an info message for the user.
Maybe it's better a LiferayLogger info message with: LiferayLogger.i(String).